site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -7,41 +7,270 @@ navcategory: admin
+
+Q: What is Maintenance Mode? +
+A: Maintenance Mode is a special admin interface for FusionAuth that helps you to interactively upgrade FusionAuth or set up database connections. You can switch to Maintenance Mode by switching to the `development` runtime mode and setting Silent Mode to false. 
+
+Q: What is Silent Mode? +
+A: link:/docs/v1/tech/guides/silent-mode[Silent Mode] is a feature of FusionAuth that automatically applies database migrations. This is useful for automated deployments or for when you want to upgrade FusionAuth without using the Maintenance Mode interface. Silent Mode can be enabled in the `development` runtime mode and as of version `1.19.0`, in `production` runtime mode too. Silent Mode will attempt to perform database migrations on startup. If the database migrations fail, FusionAuth will switch to the manual Maintenance Mode interface.
+
+Q: How many versions can I skip? +
+A: We recommended migrating from one minor version to the next. 
+
+Q: How often should I upgrade? +
+A: You can upgrade whenever you want to. It is your dependency and we won't force you. We do recommend staying relatively close to the latest release. In addition, we don't typically backport bug fixes. If you run into an issue with a bug, you'll need to upgrade to get the fix.
+
+Q: How can I find out about new releases? +
+A: We publish release notes for every release. You can find them here: link:/docs/v1/tech/release-notes[Release Notes]. 
+
+
+== Detailed Upgrade Instructions
+
+This section will guide you with detailed technical instructions on how to upgrade FusionAuth nodes on different platforms.
 
 [NOTE]
 ====
 *Readme*
 
-To upgrade to the latest version of FusionAuth using the FastPath installation go link:/docs/v1/tech/installation-guide/fast-path[here].
+To upgrade to the latest version of FusionAuth using the FastPath installation, go link:/docs/v1/tech/installation-guide/fast-path[here].
 
-To upgrade to the latest version of FusionAuth using Homebrew go link:/docs/v1/tech/installation-guide/homebrew[here].
+To upgrade to the latest version of FusionAuth using Homebrew, go link:/docs/v1/tech/installation-guide/homebrew[here].
 
-To upgrade to the latest version of FusionAuth using Docker go link:/docs/v1/tech/installation-guide/docker[here].
+To upgrade to the latest version of FusionAuth using Docker, go link:/docs/v1/tech/installation-guide/docker[here].
```

Comment:
Should we extract out the docker upgrade instructions to a shared file and include them here and there?